skbeam/core/correlation.py

```diff
@@ -405,3 +406,357 @@ def auto_corr_scat_factor(lags, beta, relaxation_rate, baseline=1):
     return beta * np.exp(-2 * relaxation_rate * lags) + baseline
 
 
+def two_time_corr(labels, images, num_frames, num_bufs, num_levels=1):
+    """Wraps generator implementation of multi-tau two time correlation
+
+    This function computes two-time correlations.
+    Original code : @author: Yugang Zhang
```

Comment:
I am not aware of `@author` meaning anything special.  Can you just have this say `Original code written by Yugang Zhang`